tasks/eva_3283/move_logs.py

```diff
@@ -0,0 +1,55 @@
+import logging
+import os
+import shutil
+from argparse import ArgumentParser
+
+from ebi_eva_common_pyutils import command_utils
+from ebi_eva_common_pyutils.logger import logging_config as log_cfg
+
+logger = log_cfg.get_logger(__name__)
+log_cfg.set_log_level(logging.INFO)
+
+
+def main():
+    argparse = ArgumentParser(description='COPY logs from clustering to logs directory')
+    argparse.add_argument('--projects_dir_path', required=True, type=str, help='The Path to projects dir')
+    argparse.add_argument('--project', required=False, type=str,
+                          help='Project for which logs needs to be copied/deleted')
+    argparse.add_argument('--copy_logs', action='store_true', default=False,
+                          help='Copy logs from clustering directory to logs directory')
+    argparse.add_argument('--delete_clustering_logs', action='store_true', default=False,
+                          help='Delete logs from clustering directory')
```

Comment:
That will work, but I can't see the rational of running `move_logs.py` script without actually copying them. 
Also you take the risk of deleting the logs before they are copied.